packages/core/src/components/Dropdown/__tests__/__snapshots__/Dropdown.snapshot.test.js.snap

```diff
@@ -4137,13 +4075,11 @@ exports[`Dropdown renders correctly without clear button 1`] = `
           onTouchEnd={[Function]}
         >
           <svg
-            aria-hidden={false}
-            className="icon clickable"
+            aria-hidden={true}
```

Comment:
Is that ok?

Reply:
Currently it has tabIndex -1 and no aria-label, so it is not focusable and should have aria-hidden